### ISSUE TITLE
make dfmt official

### ISSUE THREAD

**Issue by wilzbach** [Hackerpilot/dfmt#249]
From http://forum.dlang.org/post/nipm95$160r$1@digitalmars.com

> > From my email dated 2015/05/31 entitled "Make dformat and dfix part of the dmd distribution":
> 
> > Hi Brian,
> >
> >
> > I was very pleased at DConf to see that your toolset has nicely
> > reached maturity.
> >
> > The logical next step is to make your tools part of the dmd
> > distribution. I'm thinking making dformat and dfix trivially easy to
> > use and of great help to folks.
> >
> > What do you think? Could you work on that? Any blockers you could
> > see?
> >
> > I'm thinking getting this done for 2.068.
> >
> >
> > Thanks,
> >
> > Andrei
> 
> I have followed that with several attempts to push things forward with dfix, last on 2016/03/03, with various major contributors. There was no blocker, but also no finalization.

I guess @CyberShadow would be happy to help for the transition and @MartinNowak for inclusion in the next release?

@Hackerpilot what is blocking this?

**Comment by MartinNowak** [comment]
Yes for dfmt, rather no for dfix atm. The expectations for what dfix should be able to do are much higher, than what is possible on token or even AST base w/o running semantics. So we concluded it's better to continue using deprecations in dmd, and probably use dmd's frontend to build a dfix tool.
